picard/ui/options/scripting.py

```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.shortcut = QtWidgets.QShortcut(self.ui.script_list)
+        self.shortcut.setKey(QtGui.QKeySequence.Delete)
+        self.shortcut.activated.connect(self.script_remove_processor)
```

Comment:
Rename this to `delete_selected_script`

Reply:
@vishichoudhary  I meant the connected function. The shortcut variable itself could be `self.delete_shortcut`.